Makefile

```diff
@@ -56,8 +56,5 @@ flake8:
 pytest:
 	pytest codespell_lib
 
-pypi:
-	python setup.py sdist register upload
```

Comment:
This could definitely do with a note telling you to just release on GitHub or something....

Reply:
I don't think it would make sense to add a note here, I don't think the Makefile is a common place to look for "how to make a release" information. We could add it to the readme, but not many people actually need to know. So I created this: https://github.com/codespell-project/codespell/wiki/How-to-make-a-release